eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -34,6 +35,11 @@ class Handler implements UrlAliasHandlerInterface
      */
     const CONTENT_REPOSITORY_ROOT_LOCATION_ID = 2;
 
+    /**
+     * The maximum level of alias depth.
+     */
+    const MAX_URL_ALIAS_DEPTH_LEVEL = 60;
```

Comment:
imho this should be a setting, not a constant. Other dbs than MariaDB might allow deeper hierarchies